src/lib/util/pair.c

```diff
@@ -622,7 +622,7 @@ void fr_pair_delete_by_child_num(fr_pair_list_t *head, fr_dict_attr_t const *par
  *	- 0 on success.
  *	- -1 on failure.
  */
-int fr_pair_add_by_da(TALLOC_CTX *ctx, fr_pair_list_t *out, fr_pair_list_t *list, fr_dict_attr_t const *da)
+int fr_pair_add_by_da(TALLOC_CTX *ctx, fr_pair_t **out, fr_pair_list_t *list, fr_dict_attr_t const *da)
```

Comment:
Hm, what's going on here?

Reply:
out is just a single VP (new) being returned - so not a list head

Reply:
OK, I was just confused about the change. That's fine